Yoast/wordpress-seo
# Fix CS in lib folder

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Fixes CS issues in the CS folder.
* Un-excludes the lib folder from CS checks.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* CS should be fixed.
* Check manually for warnings. There are some remaining:
  * Code comment warning in model. The comment IS commented out code. I left it.
  * Warning about a loose comparison. This seems very intended. Integer whenever the float is loosely the same as the integer value. But I could not get the ignore to work, so I left it.
  * Warnings about using $_SERVER in the ruckusing framework runner. Not sure how to change this if at all. Leaving it like this for this PR.
* Ignored the CS error about the usage of `MYSQL_MAX_IDENTIFIER_LENGTH`. Does not seem relevant to an adapter to me. But I could be terribly wrong here.

## UI changes

## Documentation

## Quality assurance


Fixes #14996
